rockbotic_custom/models/sale_order.py

```diff
@@ -12,6 +12,9 @@ class SaleOrder(models.Model):
     street2 = fields.Char(
         'District', related="partner_shipping_id.street2", store=True)
     registration_note = fields.Text()
+    no_pa_price = fields.Float(
+        string='No PAP price', help='Price for those not belonging to the '
+        'Parents association partner')
```

Comment:
No PA price
Price for those partners that do not belong to the Parents Association